### DISCUSSION TITLE
Can't To Create Class In PIMCORE.

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#17595] [Q&A / Support] [answered]
Hi @kingjia90,

When I tried to create a new class or update a existing class and save the following error throws on my local pimcore environment and my current local pimcore environment version is - v11.3.1.1
An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'definitionModificationDate' in 'field list'

How to resolve it. Could you help me on this?

**Comment by kingjia90** [comment, marked as answer]
Hi @KasthuriAnandhan 

It was introduced in `11.3.0` by https://github.com/pimcore/pimcore/pull/15383, please make sure you ran the related migration https://github.com/pimcore/pimcore/blob/68892689cc17d299ead2dce18303db8b00da6f76/bundles/CoreBundle/src/Migrations/Version20230615103905.php#L27
